lua/entities/ent_jack_gmod_ezmachine_base.lua

```diff
@@ -147,6 +147,20 @@ if(SERVER)then
 		JMod.Hint(ply, self.ClassName)
 		return ent
 	end
+	function ENT:Initialize()
+	if(self:CustomInit())then
+		self:CustomInit()
+	end
+	if(self:GetGrade())then
+		self:SetGrade(1)
+	end
+    self.Durability=self.MaxDurability
+	--
+    self.EZupgradable=true
```

Comment:
this one should be done at the file level, not within init

Reply:
Okie, it was already defined in the base, I just moved it here.
But, will do.